appsec-kit-starter/frontend/appsec-kit/appsec-kit-plugin.ts

```diff
@@ -71,3 +71,21 @@ const plugin: DevToolsPlugin = {
     }
 };
 (window as any).Vaadin.devToolsPlugins.push(plugin);
+
+const panelConfig: PanelConfiguration = {
+    header: 'AppSec Kit',
+    expanded: true,
+    draggable: true,
+    panelOrder: 0,
+    panel: 'right',
+    floating: false,
+    tag: 'appsec-kit-plugin',
+    showOn: [Framework.Flow],
```

Comment:
```
[vite] Pre-transform error: Failed to resolve import "Frontend/generated/jar-resources/copilot/copilot-plugin-support" from "frontend/generated/jar-resources/appsec-kit/appsec-kit-plugin.js". Does the file exist?
[vite] Internal server error: Failed to resolve import "Frontend/generated/jar-resources/copilot/copilot-plugin-support" from "frontend/generated/jar-resources/appsec-kit/appsec-kit-plugin.js". Does the file exist?
  Plugin: vite:import-analysis
  File: /Users/tamas/workdir/appsec-kit/appsec-kit-demo/frontend/generated/jar-resources/appsec-kit/appsec-kit-plugin.js:3:33
  1  |  import { css as u, LitElement as d, html as f } from "lit";
  2  |  import { property as c, customElement as g } from "lit/decorators.js";
  3  |  import { Framework as h } from "Frontend/generated/jar-resources/copilot/copilot-plugin-support";
     |                                  ^
  4  |  var m = Object.defineProperty, v = Object.getOwnPropertyDescriptor, l = (e, n, o, a) => {
  5  |    for (var t = a > 1 ? void 0 : a ? v(n, o) : n, p = e.length - 1, s; p >= 0; p--)
      at formatError (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:63007:46)
      at TransformContext.error (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:63001:19)
      at normalizeUrl (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:61282:33)
      at async file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:61432:47
      at async Promise.all (index 2)
      at async TransformContext.transform (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:61353:13)
      at async Object.transform (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:63302:30)
      at async loadAndTransform (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:49045:29)
      at async viteTransformMiddleware (file:///Users/tamas/workdir/appsec-kit/appsec-kit-demo/node_modules/vite/dist/node/chunks/dep-YJaePtkC.js:58643:32)
```
I get this error when starting the demo application. I found that it is because it can’t find the `Framework`  enum.
`  3  |  import { Framework as h } from "Frontend/generated/jar-resources/copilot/copilot-plugin-support";`
If I remove the `Framework` from the import and this config to it I can start the application without error.
This might happen because the `Framework` enum is defined in a `.d.ts` file and because of this it is not available during runtime.

Reply:
While using 'flow' instead of Framework.Flow or just by removing import?

Reply:
I tried to use `'flow'`and it can't be used because `PanelConfiguration` requires `Framework`  for `showOn` property. By removing the whole `showOn: [Framework.Flow]` property it starts and I can see and open AppSec Kit in Copilot.

Reply:
Commit [fcdf57e](https://github.com/vaadin/appsec-kit/pull/152/commits/fcdf57e47fb15015ddcd79b00d161e0adf56eee6) solved this problem.